src/yb/yql/pggate/pg_txn_manager.cc

```diff
@@ -49,13 +49,18 @@ extern "C" {
 
 void YBCAssignTransactionPriorityLowerBound(double newval, void* extra) {
   txn_priority_lower_bound = ConvertBound(newval);
-  txn_priority_upper_bound = std::max(txn_priority_lower_bound, txn_priority_upper_bound)
+  txn_priority_upper_bound = std::max(txn_priority_lower_bound, txn_priority_upper_bound);
 }
 
 void YBCAssignTransactionPriorityUpperBound(double newval, void* extra) {
   txn_priority_upper_bound = std::max(txn_priority_lower_bound, ConvertBound(newval));
 }
 
+void YBCAssignTransactionPriorityLowerAndUpperBound(double newlowerbound, double newupperbound, void* extra) {
```

Comment:
What would not work with SET.

Reply:
@spolitov Would you mind explaining exactly what you mean by this? If this setter is not needed, then I can remove it. Just want some clarifications since this comment's intention is not clear.

Reply:
Those methods are not part of our API, our API is SQL.
And user could do:
```SET yb_transaction_priority_upper_bound = 0.5
SET yb_transaction_priority_lower_bound = 0.25```

That will result in the invocation of methods above.

`YBCAssignTransactionPriorityLowerAndUpperBound` is never called.

Reply:
Fair enough, will remove it then.